common/network-common/src/main/java/org/apache/spark/network/util/TransportConf.java

```diff
@@ -108,8 +108,8 @@ public int numConnectionsPerPeer() {
     return conf.getInt(SPARK_NETWORK_IO_NUMCONNECTIONSPERPEER_KEY, 1);
   }
 
-  /** Requested maximum length of the queue of incoming connections. Default -1 for no backlog. */
-  public int backLog() { return conf.getInt(SPARK_NETWORK_IO_BACKLOG_KEY, -1); }
+  /** Requested maximum length of the queue of incoming connections. Default is 64. */
+  public int backLog() { return conf.getInt(SPARK_NETWORK_IO_BACKLOG_KEY, 64); }
```

Comment:
what's the different between setting to -1 or to 64 as a default? does this change any existing behavior?

Reply:
Looks like the default previously was effectively 50 from the PR comments

Reply:
It's just being explicit, in case there are differences between various JREs. For the most common ones, at least, the default was 50.